util/dashboard/fp_004.py

```diff
@@ -53,18 +55,26 @@ def has_versioning(ontology):
 
     # retrieve version IRI or None from ontology
     version_iri = dash_utils.get_version_iri(ontology)
-    if version_iri:
-        # compare version IRI to the regex pattern
-        search = re.search(pat, version_iri)
-        if search:
-            return {'status': 'PASS'}
-        else:
-            return {'status': 'WARN',
-                    'comment': bad_format.format(version_iri)}
-    else:
+    if not version_iri:
         return {'status': 'ERROR', 'comment': missing_version}
 
-    # TODO: check that versionIRI resolves
+    # compare version IRI to the regex pattern
+    search = re.search(pat, version_iri)
+    if not search:
+        return {'status': 'WARN',
+                'comment': bad_format.format(version_iri)}
+
+    if not url_exists(version_iri):
```

Comment:
Shouldnt this check come first? Else we will end up with a warning when the pattern matches, but the IRI does not resolve?

Reply:
either way is fine, just tell me what you want and I will do it

Reply:
Updated in ed4ef6c

Reply:
Yeah, not resolvable trumps bad format. Thanks!